zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -109,14 +109,14 @@ def get_entities(self) -> list:
             self.kube_client, self.cluster_id, self.alias, self.environment, self.region, self.infrastructure_account,
             namespace=self.namespace)
 
-        postgresql_cluster_entities = get_postgresql_clusters(
+        postgresql_cluster_entities, pce = itertools.tee(get_postgresql_clusters(
```

Comment:
Is this a more fancy way to say `list(...)`?

Reply:
This keeps the generatorness of the generator (well, both of them), so we don't have a mix of list and generator based logic.

Reply:
Yeah, I see the extra "smartness" now...